doc/getting-started.md

```diff
@@ -104,6 +104,18 @@ docker run -it -p 8080:3000 \
 The last command will make Convos available on http://localhost:8080, and
 persist data in `$HOME/convos/data`.
 
+To expose Convos to localhost only, prepend `127.0.0.1` like so:
+
+```bash
+-p 127.0.0.1:8080:3000
```

Comment:
I think this is a bit too much information here. I'm not entirely sure how frequent people start Convos on localhost only.

Reply:
I tend to agree with jhthorsen about this, and it's not really convos specific info.

Reply:
I felt like this might of been the case, I did feel the same way a bit. I've simply removed the commit.